_delphi_utils_python/delphi_utils/validator/static.py

```diff
@@ -166,8 +166,6 @@ def _get_valid_geo_values(self, geo_type):
         gmpr = GeoMapper()
         valid_geos = gmpr.get_geo_values(geomap_type)
         valid_geos |= set(self.params.additional_valid_geo_values.get(geo_type, []))
-        if geo_type == "county":
-            valid_geos |= set(x + "000" for x in gmpr.get_geo_values("state_code"))
```

Comment:
lolololol, we shoulda just been using `_get_valid_geo_values()` instead of the [`geo_type_translator`](https://github.com/cmu-delphi/delphi-epidata/blob/b9cd48bb8a4e8fb14a6f8408bcaa4fd0e1a394ea/src/server/_params.py#L62)